carma-messenger-core/carma-messenger/launch/carma-messenger.launch.py

```diff
@@ -72,15 +63,10 @@ def generate_launch_description():
         ]
     )
 
-    v2x_group = GroupAction(
-        actions=[
-            IncludeLaunchDescription(
-                PythonLaunchDescriptionSource([get_package_share_directory('v2x-ros-conversion'), '/v2x-ros-conversion.launch.py']),
-                launch_arguments = {
-                    'configuration_delay' : [configuration_delay]
```

Comment:
Is it possible this didn't work before because `configuration_delay` is in brackets? It might not make a difference, but not quite sure

Reply:
Possible, seems like the plugins group also passes this parameter with the brackets. Not able to recreate the issue locally but seems like the param value can be passed without brackets as well so will test on vehicle

Reply:
Good point! I had missed that. Yes, agreed that probably won't have an effect then